src/commands/install-sfdx.yml

```diff
@@ -8,32 +8,64 @@ description: >
+    description: 'By default, the latest version of the standalone CLI will be installed on linux. To install via npm, supply a version tag such as "latest" or "6".  For windows, only npm installation is supported and omitting the version will give you "latest"'
     default: ''
     type: string
+  os:
+    description: operating system to run tests on
+    type: enum
+    enum: ['linux', 'windows']
+    default: 'linux'
 
+# there are four combos of OS and version
 steps:
-  # when version is defined.
   - when:
-      condition: << parameters.version >>
+      condition:
+        and:
+          # windows installer has a UI.  so we install node version
+          - equal: ['windows', <<parameters.os>>]
+          - <<parameters.version>>
       steps:
-        - run:
-            name: Install SFDX - NPM
-            command: |
-              command -v npm >/dev/null 2>&1 || { echo >&2 "npm not installed in the current environment.  Aborting."; exit 1; }
-              npm install sfdx-cli@<<parameters.version>> --global
-  # unless version is _not_ defined.
-  - unless:
-      condition: <<parameters.version>>
+        run:
+          name: Install SFDX - NPM <<parameters.version>> on windows
+          command: |
+            npm install sfdx-cli@<<parameters.version>> --global
+  - when:
+      condition:
+        and:
+          - equal: ['windows', <<parameters.os>>]
+          - not: <<parameters.version>>
+      steps:
+        run:
+          name: Install SFDX - NPM latest on windows
+          # if no version, use the latest
+          command: |
+              npm install sfdx-cli --global
+  - when:
+      condition:
+        and:
+          - equal: ['linux', <<parameters.os>>]
+          - <<parameters.version>>
+      steps:
+        run:
+          name: Install SFDX - NPM on linux @<<parameters.version>>
+          command: |
+            command -v npm >/dev/null 2>&1 || { echo >&2 "npm not installed in the current environment.  Aborting."; exit 1; }
+            npm install sfdx-cli@<<parameters.version>> --global
+  - when:
+      condition:
+        and:
+          - equal: ['linux', <<parameters.os>>]
+          - not: <<parameters.version>>
       steps:
-        - run:
-            name: Install SFDX - Standalone
-            command: |
-              cd /tmp
-              wget https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz
-              mkdir sfdx
-              tar xJf sfdx-linux-amd64.tar.xz -C sfdx --strip-components 1
-              ./sfdx/install
+        run:
+          name: Install SFDX - Standalone on linux
+          command: |
+            cd /tmp
+            wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable-rc/sfdx-linux-x64.tar.xz
+            mkdir sfdx
+            tar xJf sfdx-linux-amd64.tar.xz -C sfdx --strip-components 1
```

Comment:
have you tested this? I'd expect that the file name to be different after updating the url